lines/transformer/transform.py

```diff
@@ -0,0 +1,78 @@
+"""
+Rundown API response transformer mechanism for creating temporal updates.
+
+Generator of temporal records for `lines.producer.creator`
+
+(e.g.)
+    Temporal records represented by key: value pairs
+    NOTE: this must contain an temporal element
+    in this case the field: "event_date" from API response
+
+    "timestamp": "2020-02-02T23:30:00Z" is referenced as temporal element
+
+    {
+        "key": {
+            "eventId": "9a35b8986a76ea364be331cb453ec",  # string
+            "sportId": 2  # integer
+        },
+        "data": {
+            "timestamp": "2020-02-01T17:22:12Z",
+            "eventDate": "2020-02-02T23:30:00Z",
+            ...
+        }
+    }
+"""
+import datetime
+import json
+from typing import Union
+
+from prefect import Task
+
+from lines.libs.facades import LinesEventFacade
+from lines.libs.errors import MalformedKeyError
+
+
+class RundownTransformer(Task):
+    """RundownTransformer object."""
+
+    def __init__(self):
+        """Constructor for RundownTransformer."""
+        self.log_level = "INFO"
+        super().__init__()
+
+    def run(self, record: LinesEventFacade) -> Union[dict, None]:
+        """
+        Generate temporal record for data store.
+
+        Args:
+            record: (LinesEventFacade) - input to generate temporal record for
```

Comment:
Let's stick to a consistent docstring pattern. If you use Google, use it as [such](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

Reply:
from google docstring documentation:
```
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """
```

Reply:
Yes.. hence:

```
record (LinesEventFacade): Input to generate temporal record.
```